BLSQ/iaso
# POLIO-1627 Chronogram: allow to filter by campaign

Allow to filter chronograms by campaign.

Related JIRA tickets : [POLIO-1627](https://bluesquare.atlassian.net/browse/POLIO-1627)

## Changes

- allow the backend API to filter chronograms by campaign
- expose the list of campaigns used in chronograms via an OPTIONS endpoint
    - rename `useOptionChronogramTask.ts` to `useOptionChronogram.ts` in the frontend
- allow the frontend to filter chronograms by campaign

## How to test

1. go to `http://localhost:8081/dashboard/polio/chronogram/`
2. ensure you have at least 2 chronograms related to two different campaigns
3. try to filter by campaign

## Video

https://github.com/user-attachments/assets/6d5bb950-b643-4abe-b620-ca37481154b8


[POLIO-1627]: https://bluesquare.atlassian.net/browse/POLIO-1627?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
